lib/success.js

```diff
@@ -65,21 +65,81 @@ export default async function success(pluginConfig, context, { Octokit }) {
     const releaseInfos = releases.filter((release) => Boolean(release.name));
     const shas = commits.map(({ hash }) => hash);
 
-    const searchQueries = getSearchQueries(
-      `repo:${owner}/${repo}+type:pr+is:merged`,
-      shas,
-    ).map(
-      async (q) =>
-        (await octokit.request("GET /search/issues", { q })).data.items,
-    );
+    const loadAssociatedPRsQuery = (shas) => `#graphql
+      query getAssociatedPRs($owner: String!, $repo: String!) {
+        repository(owner: $owner, name: $repo) {
+          ${shas.map((sha) => {
+            return `"${sha.slice(0, 6)}": object(oid: ${sha}) {
+              ...on Commit {
+                associatedPullRequests(first: 100) {
+                  nodes {
+                    url
+                    number
+                  }
+                }
+              }
+            }`
+          }).join("")}
+        }
+      }
+    `;
+
+    // MOCK ONLY
+    // const mockReponse = {
+    //   "data": {
+    //     "repository": {
+    //       "com1": {
+    //         "associatedPullRequests": {
+    //           "nodes": [
+    //             {
+    //               "number": 2
+    //             }
+    //           ]
+    //         }
+    //       },
+    //       "com2": {
+    //         "associatedPullRequests": {
+    //           "nodes": [
+    //             {
+    //               "number": 1
+    //             }
+    //           ]
+    //         }
+    //       }
+    //     }
+    //   }
+    // };
+    // console.log(somedata)
 
-    const searchQueriesResults = await Promise.all(searchQueries);
-    const uniqueSearchQueriesResults = uniqBy(
-      flatten(searchQueriesResults),
+    const { data } = await octokit.graphql(loadAssociatedPRsQuery(shas), { owner, repo });
+    
+    const associatedPRs = toArray(mockReponse.data.repository).map((item) => item.associatedPullRequests.nodes);
```

Comment:
Following the the GQL query run which (consumes the `loadAssociatedPRsQuery`), the deduced data `associatedPRs` is exactly structurally like the commented out `searchQueriesResults` data above there.